docs/QUICKSTART.md

```diff
@@ -1,76 +1,195 @@
-
-Example:
-```bash
-cat <<EOF | kubectl apply -f -
-apiVersion: eraser.sh/v1alpha1
-kind: ImageList
-metadata:
-  name: imagelist
-spec:
-  images:
-    - sha256:2834dc507516af02784808c5f48b7cbe38b8ed5d0f4837f16e78d00deb7e7767
-    - docker.io/library/nginx:latest
-    - redis
-EOF
-```
+# Quickstart Tutorial
+
+This tutorial demonstrates the functionality of Eraser and validates that non-running images are removed after applying an `ImageList` with the declared images.
+
+1. Create a cluster
+
+  Eraser can be deployed to any Kubernetes cluster, however, for the purposes of this tutorial a `Kind` cluster will be created with a control-plane and two worker nodes.
+
+    ``` shell
+    # get kind
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.13.0/kind-linux-amd64
+    chmod +x ./kind
+    sudo mv ./kind /usr/bin
+    ```
+    
+    ```shell 
+    # create a cluster
+    cat <<EOF | kind create cluster --config -
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    EOF
+    ``` 
+  
+    ```shell
+    # Verify your cluster is running.
+    $ kubectl cluster-info --context kind-kind
+    Kubernetes control plane is running at https://127.0.0.1:40989
+    CoreDNS is running at https://127.0.0.1:40989/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+    To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+    ``` 
+
+2. Deploy Eraser to the Cluster
+
+    ```bash
+    $ kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v0.1.0/deploy/eraser.yaml
+    namespace/eraser-system created
+    customresourcedefinition.apiextensions.k8s.io/imagejobs.eraser.sh created
+    customresourcedefinition.apiextensions.k8s.io/imagelists.eraser.sh created
+    serviceaccount/eraser-controller-manager created
+    role.rbac.authorization.k8s.io/eraser-leader-election-role created
+    clusterrole.rbac.authorization.k8s.io/eraser-manager-role created
+    rolebinding.rbac.authorization.k8s.io/eraser-leader-election-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/eraser-manager-rolebinding created
+    configmap/eraser-manager-config created
+    deployment.apps/eraser-controller-manager created
+    ```
+
+    ```bash
+    $ kubectl get pods -n eraser-system
+    NAME                                         READY   STATUS    RESTARTS   AGE
+    eraser-controller-manager-759c7c8794-q8qqg   1/1     Running   0          22s
+    ```
+
+3. Apply and delete a deployment
+
+    Applying and deleting a deployment will leave unused images on the nodes where it was deployed. For this example, an `ngnix` deployment will be used. In following steps, these images will be used to verify that Eraser is removing the correct images.
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps/v1
+    kind: Deployment
```

Comment:
should this be a Daemonset? to make sure we get an image in each node

Reply:
updated to DaemonSet in [83eaf48](https://github.com/Azure/eraser/pull/186/commits/83eaf483c7b3f4e7463e0f3a438e51c3ddaed5f9)